src/Model/Item.php

```diff
@@ -1065,9 +1065,6 @@ public static function deleteForUser($condition, $uid)
 			// "Deleting" global items just means hiding them
 			if ($item['uid'] == 0) {
 				DBA::update('user-item', ['hidden' => true], ['iid' => $item['id'], 'uid' => $uid], true);
-
-				// Delete notifications
-				DBA::delete('notify', ['iid' => $item['id'], 'uid' => $uid]);
```

Comment:
Why did you remove this code?

Reply:
We have got a foreign key on this table. So deleting the corresponding `item-uri` will automatically delete this entry as well.